lib/grape/swagger/jsonapi/resources.rb

```diff
@@ -1,15 +1,17 @@
-require "grape-swagger"
+require 'grape-swagger'
 
 # Only require the bits of Rails that jsonapi-resources uses
-require "action_controller"
-require "rails"
+require 'action_controller'
```

Comment:
Can we not require rails at all? It would be nice to be able to build APIs in standalone grape for smaller applications without having to load all of rails.

Reply:
I wanted that too, but part of the JSONAPI resources gem uses some Rails controller code. Part of making this into a standalone gem was wanting to make some non-rails specs in order to do that, but there's not been time to vendor the bits that are used and extract this.